tests/foreign_object/test_tuple_lookups.py

```diff
@@ -63,9 +63,11 @@ def test_exact(self):
                 )
 
     def test_exact_subquery(self):
-        with self.assertRaisesMessage(
-            ValueError, "'exact' doesn't support multi-column subqueries."
-        ):
+        msg = (
+            "The QuerySet value for the exact lookup must have 2 selected "
+            "fields (received 1)"
+        )
+        with self.assertRaisesMessage(ValueError, msg):
```

Comment:
we can change this test to a positive test case

Reply:
I think the PR already include some that do pass with or without multiple fields selected.